layers/+emacs/org/packages.el

```diff
@@ -184,7 +186,7 @@ Will work on both org-mode and any mode that accepts plain html."
 
       (dolist (prefix '(
                         ("mb" . "babel")
-                        ("mC" . "clocks")
+                        ("mC" . "clocks/contacts")
```

Comment:
Would it make any sense to add `/contacts` conditionally?

Reply:
I agree. This should be added only when `org-contacts` is enabled.